### ISSUE TITLE
test depends on Plots

### ISSUE THREAD

**Issue by mgkuhn** [JuliaComputing/Xyce.jl#5]
There probably should be a `test/Projects.toml` to indicate a dependency on `Plots.jl`:

```
(@v1.6) pkg> test Xyce
     Testing Xyce
      Status `/tmp/jl_TUYOWc/Project.toml`
  [1f15a43c] CxxWrap v0.11.2
  [1b67bd2a] Xyce v0.1.0
  [11346573] XyceWrapper_jll v0.1.1+0
      Status `/tmp/jl_TUYOWc/Manifest.toml`
  [1f15a43c] CxxWrap v0.11.2
  [692b3bcd] JLLWrappers v1.3.0
  [1914dd2f] MacroTools v0.5.6
  [21216c6a] Preferences v1.2.2
  [1b67bd2a] Xyce v0.1.0
  [f5851436] FFTW_jll v3.3.9+8
  [656ef2d0] OpenBLAS32_jll v0.3.12+1
  [ca45d3f4] SuiteSparse32_jll v5.8.1+0
  [b6fd3212] Trilinos_jll v12.12.1+1
  [11346573] XyceWrapper_jll v0.1.1+0
  [b812464d] Xyce_jll v7.3.1+0
  [3eaa8342] libcxxwrap_julia_jll v0.8.6+0
  [0dad84c5] ArgTools `@stdlib/ArgTools`
  [56f22d72] Artifacts `@stdlib/Artifacts`
  [2a0f44e3] Base64 `@stdlib/Base64`
  [ade2ca70] Dates `@stdlib/Dates`
  [f43a241f] Downloads `@stdlib/Downloads`
  [b77e0a4c] InteractiveUtils `@stdlib/InteractiveUtils`
  [b27032c2] LibCURL `@stdlib/LibCURL`
  [76f85450] LibGit2 `@stdlib/LibGit2`
  [8f399da3] Libdl `@stdlib/Libdl`
  [56ddb016] Logging `@stdlib/Logging`
  [d6f4376e] Markdown `@stdlib/Markdown`
  [ca575930] NetworkOptions `@stdlib/NetworkOptions`
  [44cfe95a] Pkg `@stdlib/Pkg`
  [de0858da] Printf `@stdlib/Printf`
  [3fa0cd96] REPL `@stdlib/REPL`
  [9a3f8284] Random `@stdlib/Random`
  [ea8e919c] SHA `@stdlib/SHA`
  [9e88b42a] Serialization `@stdlib/Serialization`
  [6462fe0b] Sockets `@stdlib/Sockets`
  [fa267f1f] TOML `@stdlib/TOML`
  [a4e569a6] Tar `@stdlib/Tar`
  [cf7118a7] UUIDs `@stdlib/UUIDs`
  [4ec0a83e] Unicode `@stdlib/Unicode`
  [e66e0078] CompilerSupportLibraries_jll `@stdlib/CompilerSupportLibraries_jll`
  [deac9b47] LibCURL_jll `@stdlib/LibCURL_jll`
  [29816b5a] LibSSH2_jll `@stdlib/LibSSH2_jll`
  [c8ffd9c3] MbedTLS_jll `@stdlib/MbedTLS_jll`
  [14a3606d] MozillaCACerts_jll `@stdlib/MozillaCACerts_jll`
  [4536629a] OpenBLAS_jll `@stdlib/OpenBLAS_jll`
  [bea87d4a] SuiteSparse_jll `@stdlib/SuiteSparse_jll`
  [83775a58] Zlib_jll `@stdlib/Zlib_jll`
  [8e850ede] nghttp2_jll `@stdlib/nghttp2_jll`
  [3f19e933] p7zip_jll `@stdlib/p7zip_jll`
     Testing Running tests...
ERROR: LoadError: ArgumentError: Package Plots not found in current path:
- Run `import Pkg; Pkg.add("Plots")` to install the Plots package.

Stacktrace:
 [1] require(into::Module, mod::Symbol)
   @ Base ./loading.jl:893
 [2] include(fname::String)
   @ Base.MainInclude ./client.jl:444
 [3] top-level scope
   @ none:6
in expression starting at /local/bigscratch/mgk25/julia-depot/packages/Xyce/qhfY7/test/runtests.jl:2
ERROR: Package Xyce errored during testing
```
Tested with Julia 1.6.2 on Ubuntu 20.04.

**Comment by pepijndevos** [comment]
Yes. Honestly the tests don't really actually test anything at the moment. The better solution would be to make the tests not depend on plots and actually test the results are as expected. I'd accept a PR that does either though.

**Comment by ViralBShah** [comment]
This needs to be resolved for #3

**Comment by ViralBShah** [comment]
@GlenHertz Close?

**Comment by GlenHertz** [comment]
This is fixed by PR #8.
